torch/csrc/api/include/torch/nn/options/pixelshuffle.h

```diff
@@ -21,6 +21,20 @@ struct TORCH_API PixelShuffleOptions {
   TORCH_ARG(int64_t, upscale_factor);
 };
 
+/// Options for the `PixelUnshuffle` module.
+///
+/// Example:
+/// ```
+/// PixelUnshuffle model(PixelUnshuffleOptions(5));
+/// ```
+struct TORCH_API PixelUnshuffleOptions {
+  PixelUnshuffleOptions(int64_t downscale_factor)
```

Comment:
nit: for single arg unique constructor, we normally put explicit before, this can avoid warnings in phabricator as well.

Reply:
Thanks for catching this! I might be wrong, but it looks like the implicit behavior is desired for C++ module options. None of the option structs defined in `torch/csrc/api/include/torch/nn/options` use `explicit`, and in fact several have the `/* implicit */` comment. Further, the C++ module tests are often written to use the implicit conversion. So to follow our standards and match these, I think I should add a similar `/* implicit */` comment here as well. Thoughts on this?

Reply:
@jbschlosser 
The C++ lib is contributed by many external contributors. Multiple coding styles are mixed together at the moment.
I plan to standardized it a bit in the future.
Adding /*implicit*/ is fine, I will update them in one shot later.